resources/pathway2Reactome_mappings.tsv

```diff
@@ -0,0 +1,15 @@
+pathway	target	reactomeId	description
+Androgen	AR	R-HSA-8940973	RUNX2 regulates osteoblast differentiation
+EGFR	EGFR	R-HSA-8856828	Clathrin-mediated endocytosis
+Estrogen	ESR1	R-HSA-8939902	Regulation of RUNX2 expression and activity
+Hypoxia	HIF1A	R-HSA-123456	XXX Pathway desc to be updated

```

Comment:
Can the description be updated? `R-HSA-123456` seems to be a placeholder and I couldn't easily find an alternative match. If not, should it be removed?

Reply:
This sort of errors were previously described in #713 and have yet to be addressed. I'm making a comment in the PR for clarification.